integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -439,6 +426,60 @@ def export(self, **kwargs):  # pylint: disable=R0915
             enterprise_customer_identifier=self.enterprise_customer.name
         )
 
+    def _filter_out_pre_transmitted_enrollments(
+            self,
+            enrollment_queryset,
+            channel_name,
+            grade,
+            transmission_audit
+    ):
+        """
+        Given an enrollment_queryset, returns only enrollments that are not already transmitted
+        """
+        included_enrollments = set()
+        for enterprise_enrollment in enrollment_queryset:
+            if transmission_audit and \
+                    is_already_transmitted(transmission_audit, enterprise_enrollment.id, grade):
+                # We've already sent a completion status for this enrollment
+                generate_formatted_log(
+                    'Skipping export of previously sent enterprise enrollment. '
+                    'EnterpriseEnrollment: {enterprise_enrollment_id}'.format(
+                        enterprise_enrollment_id=enterprise_enrollment.id
+                    ),
+                    channel_name=channel_name,
+                    enterprise_customer_identifier=self.enterprise_customer.name
+                )
+                continue
+            included_enrollments.add(enterprise_enrollment)
+        return included_enrollments
+
+    def create_enrollment_queryset(self, learner_to_transmit, course_run_id, channel_name):
+        """
+        Creates queryset of EnterpriseCourseEnrollments ordered by course_id
+        Queryset is filtered by learner and course_run_id if both are provided
+        """
+        enrollment_queryset = EnterpriseCourseEnrollment.objects.select_related(
+            'enterprise_customer_user'
+        ).filter(
+            enterprise_customer_user__enterprise_customer=self.enterprise_customer,
+            enterprise_customer_user__active=True,
+        )
+        if learner_to_transmit and course_run_id:
+            enrollment_queryset = enrollment_queryset.filter(
+                course_id=course_run_id,
+                enterprise_customer_user__user_id=learner_to_transmit.id,
+            )
+            generate_formatted_log(
+                'Exporting single learner. Course: {course_run}, User: {user_id}'.format(
```

Comment:
If this is the lms_user_id can you label it as the LMS User Id so that looking at logs later no one has to go to the code to look up what the number means?